tests/tests.py

```diff
@@ -1071,3 +1071,16 @@ def test_improper_configuration_raises(self):
             "REFERRER_POLICY",
             "invalid",
         )
+
+
+class UnicodeDataTests(TestCase):
```

Comment:
(Minor)

This class can be written better for more readability and better reuse in the future (but it is up to you since it is minor):

```
class UnicodeDataTests(TestCase):
    USERNAME = "ñoñó1234"
    IP_ADDRESS = "127.0.0.1"

    def test_unicode_data_in_cache_key(self):
        self._execute_with_unicode_data(self.USERNAME, self.IP_ADDRESS)

    def _execute_with_unicode_data(self, username, ip):
        try:
            increment_counters(username=username, ip=ip)
            reset_counters(username=username, ip=ip)
            throttling_delay(username=username, ip=ip)
            attempt_count(attempt_type="auth", id=username)
        except Exception:
            self.fail("Unicode data not allowed")
```

Reply:
alright, I think it got better. Let me change that.